### DISCUSSION TITLE
a problem while uploading a video ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [requarks/wiki#5469] [Error / Bug Report] [answered]
Hello , 

I've tried many times to upload a video and every time I receive an error says " upload Video failed "

I've increased the size and still didn't work !

IS there any solution ? 

Best Regards

**Comment by NGPixel** [comment, marked as answer]
Don't upload videos (or very large files) to Wiki.js. Use a proper object store (AWS S3, Azure Blob Storage, DigitalOcean Spaces, etc.) instead.

**Reply by lukemoreira** [reply]
Hi, I am using an Azure blob but having problems uploading anything over 5MB.
I tested the blob and it will take files up to 100TB.

Any Idea? My Wiki is deployed from a docker image to an Azure Web App.

Thanks.

**Reply by NGPixel** [reply]
@lukemoreira You need to increase the upload limit under Admin > Security.
